config.txt

```diff
@@ -0,0 +1 @@
+100
```

Comment:
That is used to keep values related to the configuration such as purging age of an old bundle etc..